R/use_github_file.R

```diff
@@ -76,6 +76,11 @@ use_github_file <- function(repo_spec,
     ref = ref,
     host = host
   )
+  if (!is.null(attr(lines, "error"))) {
```

Comment:
This doesn't feel quite right to me, because generally errors should bubble up by default. If you don't want to error, _then_ you need to do something special.